server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -784,7 +784,14 @@ public Consumer createConsumerFromDTO(ConsumerDTO consumer, ConsumerType type, P
         }
 
         Consumer consumerToCreate = new Consumer();
-        consumerToCreate.setOwner(owner);
+
+
+        log.debug("SETTING OWNER ID FROM OWNER: {} => {}", owner, owner.getId());
```

Comment:
Can we either delete these or un-all-caps them?

Reply:
Removed